src/main/java/internet/shop/Demo.java

```diff
@@ -40,82 +44,92 @@ public static void main(String[] args) {
+        System.out.println(driverService.delete(2L));
+        System.out.println(driverService.delete(3L));
+        System.out.println(driverService.getAll());
+
+        System.out.println(driverService.create(driver1));
+        System.out.println(driverService.create(driver2));
+        driver3.setId(5L);
+        System.out.println(driverService.update(driver3));
+        System.out.println(driverService.getAll());
+        System.out.println("=============================================");
         Car car1 = new Car();
         car1.setDrivers(driver1);
-        car1.setId(1L);
         car1.setManufacturer(manufacturer);
         car1.setModel("Audi");
 
         Car car2 = new Car();
         car2.setDrivers(driver1);
         car2.setDrivers(driver2);
-        car2.setId(2L);
         car2.setManufacturer(manufacturer2);
         car2.setModel("BMW");
 
         Car car3 = new Car();
         car3.setDrivers(driver1);
         car3.setDrivers(driver2);
         car3.setDrivers(driver3);
-        car3.setId(3L);
         car3.setManufacturer(manufacturer3);
         car3.setModel("Mercedes");
 
         CarService carService = (CarService) injector
                 .getInstance(CarService.class);
 
-        carService.create(car1);
-        carService.create(car2);
-        carService.create(car3);
-        printResultManufacturerCar(carService);
-
-        carService.delete(1L);
-        carService.delete(2L);
-        carService.delete(3L);
-        printResultManufacturerCar(carService);
-
-        carService.create(car2);
-        printResultManufacturerCar(carService);
-        car3.setId(4L);
-        car3.setModel("-");
-        carService.update(car3);
-        printResultManufacturerCar(carService);
-        System.out.println("======================================================");
-        DriverService driverService = (DriverService) injector
-                .getInstance(DriverService.class);
-
-        driverService.create(driver1);
-        driverService.create(driver2);
-        driverService.create(driver3);
-        printResultManufacturerDriver(driverService);
-
-        driverService.delete(1L);
-        driverService.delete(2L);
-        driverService.delete(3L);
-        printResultManufacturerDriver(driverService);
-
-        driverService.create(driver2);
-        printResultManufacturerDriver(driverService);
-        driver3.setId(4L);
-        driver3.setLicenceNumber("000");
-        driverService.update(driver3);
-        printResultManufacturerDriver(driverService);*/
+        System.out.println(carService.create(car1));
+        System.out.println(carService.update(car1));
+        System.out.println(carService.get(1L));
+        System.out.println(carService.create(car2));
+        System.out.println(carService.update(car2));
+        System.out.println(carService.get(2L));
+        System.out.println(carService.create(car3));
+        System.out.println(carService.update(car3));
```

Comment:
let's change the state of the car before updating it - change model, for example

Reply:
done